openstack/nova/files/mitaka/nova+virt+libvirt+driver.py

```diff
@@ -4248,7 +4248,7 @@ def _create_serial_console_devices(self, guest, instance, flavor,
                     CONF.serial_console.proxyclient_address)
                 console.listen_port = (
                     serial_console.acquire_port(
-                        console.listen_host))
+                        "::1"))
```

Comment:
Our kvm script actually has kvm.real listen permanently on ::1. The telnet_front-executed socats aren't permanent. If nova-compute is restarted, it looks up all known ports, and it may hit one for which the socat isn't running now.